GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -288,7 +288,11 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
     }
     
     func cameraDidAppear(_ viewController: CameraViewController) {
-        showOnboardingIfNeeded()
+        if shouldShowOnBoarding() {
```

Comment:
What do you think of showing it after the onboarding?

Reply:
I was thinking about it, but I haven't decided on the flow yet. We should take a look on this before the release, since there are different possibilities regarding toolTips/dialogs...